string-manipulation-3/src/reverse-words.js

```diff
@@ -0,0 +1,12 @@
+/* exported reverseWords */
+// split string into array and assign it to variable stringOne;
+// reverse that array;
+// .join to turn it back into string;
+// .split it back into an array as backwards strings;
+// reverse it again and join it back into a string;
+function reverseWords(string) {
+
+  var stringOne = string.split('').reverse().join('').split(' ').reverse().join(' ');
```

Comment:
I think this works (if the tests pass, it must!), but seems more complex than it needs to be.

Couldn't you just grab all the words (split on space), then reverse the letters in each word and then put the words back together again?

So, on the one hand, this is a clever solution that accomplishes the task all in one statement. On the other hand, it takes a lot of effort to figure out what it's doing and to confirm that it works.